estimagic/logging/create_database.py

```diff
@@ -143,7 +144,14 @@ def prepare_database(
         to the database can be accessed via ``database.bind``.
 
     """
-    db_options = {} if db_options is None else db_options
+    standard_dash_options = {"no_browser": False, "port": None, "rollover": 500}
```

Comment:
This should be part of the normal arguments processing for the optimization

Reply:
You are right. Moved in c56364a71dbbaf0122f4f136ffaedbeaa49b65cd.